libfreerdp/codec/test/TestFreeRDPRegion.c

```diff
@@ -632,7 +632,7 @@ static int test_norbert_case() {
 		0, 0, 1920, 1080
 	};
 	RECTANGLE_16 expected_inter_extents = {
-		0, 0, 1920, 1078
+		2, 0, 1920, 1078
 	};
```

Comment:
I found the expected result is not correct when this test report failure.
The code above this line is:

    RECTANGLE_16 inRectangles[5] = {
            {1680,    0, 1920,  242},
            { 294,  242,  971,  776},
            {1680,  242, 1920,  776},
            {1680,  776, 1920, 1036},
            {   2, 1040,   53, 1078}
    };
=======================
The 'union' of above rects should be 2,0,1920,1078 
=======================
    RECTANGLE_16 screenRect = {
        0, 0, 1920, 1080
    };

=======================
Then intersect with screenRect 0,0,1920.,1080, the result should be 2,0,1920,1078 
=======================
    RECTANGLE_16 expected_inter_extents = {
        2, 0, 1920, 1078
    };

So I think the expected rect should be fixed